examples/cross_build/tricore.rst

```diff
@@ -0,0 +1,51 @@
+.. _example_cross_build_tricore:
+
+
+Building packages with Tricore compiler
+=======================================
+
+.. include:: ../../common/experimental_warning.inc
+
+
+Tricore compiler is a popular compiler for embedded in some domains as automotive.
```

Comment:
```suggestion
Tricore compiler is a popular compiler for embedded in some domains such as automotive.
```

@photex, you can directly propose changes like this, clicking in the +- icon above, so it is possible to accept them with one click.